### PR TITLE
Enhanced output for performance testing rig

### PR DESCRIPTION
**Pull request by rzats**
**Prerequisites**:

- [x] Unless it is a documentation hotfix it should be merged against the `dev` branch
- [x] Branch is up-to-date with the branch to be merged with, i.e. `dev`
- [x] Build is successful
- [x] Code is cleaned up and formatted

### Summary

Updates the performance testing rig to make its PR comment output a summary of the run, which looks like this:

Screenshot 2023-02-27 at 21 33 08

### CODE REVIEW

**Comment by rzats** [comment]
Unsure if this will trigger the newer or older version of the GitHub Action, but:

`/run performance test`

**Review by krivard — APPROVED** [review]
👍 Can merge as-is; question identifies a possible opportunity to reduce some coupling

File: .github/workflows/performance-tests.yml [hunk 1]
```diff
@@ -69,20 +77,78 @@ jobs:
...
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              body: `✅ Performance tests complete! Result summary:
+              - Total requests: **${{ env.REQUEST_COUNT }}**
+              - Total failures: **${{ env.FAILURE_COUNT }}**
+              - Min response time: **${{ env.MIN_TIME }} ms**
+              - Max response time: **${{ env.MAX_TIME }} ms**
+              - Average response time: **${{ env.AVG_TIME }} ms**
+              - Median response time: **${{ env.MED_TIME }} ms**
+              - Requests per second: **${{ env.REQUESTS_PER_SEC }}**
```
[the remark this answers is not in the document]
Comment (rzats) [reply .github/workflows/performance-tests.yml]: I tried grabbing them directly from the `outputs` and that worked - thanks for the pointer!

**Comment by rzats** [comment]
@krivard requesting a re-review since I applied the change that reduced some repetitive code

**Review by krivard — APPROVED** [review]
👍 Solid!

**Pull request merged**